pkg/lvm/lvm_util.go

```diff
@@ -265,6 +266,38 @@ func buildLVMDestroyArgs(vol *apis.LVMVolume) []string {
 	return LVMVolArg
 }
 
+// buildLVMRestoreThinSnapshotArgs returns lvcreate command for the retore volume from thin snapshot
+func buildLVMRestoreThinSnapshotArgs(vol *apis.LVMVolume) []string {
+	var LVMRestoreThinVolArg []string
+
+	// command to create restored thin volume from thin snapshot
+	// `lvcreate -s -n <restore-volume-name>  lvmvg/<thin-snapshot-id>`
```

Comment:
Is this more suitable?

```suggestion
	// `lvcreate -s -n <restore-volume-name>  lvmvg/<thin-snapshot-name>`
```